shell/opslevel/upload_prismaci.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+#
+# This script uploads Twistlock Image Scan results (in a compact format) to Prisma Cloud.
+# This is not a standalone script - it is meant to be called from the upload.sh script only.
+#
+DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
+
+IMAGE_SCAN_ARTIFACT_NAME="tmp/test-results/image_scan.json"
+SCAN_RESULTS_FILE="/tmp/image_scan.json"
+SCAN_RESULTS_OPSLEVEL_FILE="/tmp/image_scan_opslevel.json"
+OPSLEVEL_PRISMACI_CUSTOM_EVENT_ID="625cd47f-257a-4ed4-ac56-7efb4dc25158"
```

Comment:
This seems specific to Outreach, we should change this if possible

Reply:
Also if this is a secret value at all, it'll need to be rotated now

Reply:
The UUID is not a secret - and it cannot be rotated since it is tight to an 'extension' with lots of config on opslevel side. But it is a good habit still to move it out to config and 'hide' it from general public since the uploads to the extension in Opslevel are unauthenticated.